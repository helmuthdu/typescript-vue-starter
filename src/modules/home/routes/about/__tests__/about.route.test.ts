import { createLocalVue, shallowMount, Wrapper } from '@vue/test-utils';
import Vuetify from 'vuetify';
import AboutRoute from '../about.route.vue';

const localVue = createLocalVue();

localVue.use(Vuetify);

describe('Route -> About', () => {
  let wrapper: Wrapper<AboutRoute>;

  beforeEach(() => {
    wrapper = shallowMount(AboutRoute, { localVue });
  });

  it('should renders with props', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
