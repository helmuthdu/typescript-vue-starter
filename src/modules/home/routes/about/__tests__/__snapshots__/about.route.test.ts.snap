// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route -> About should renders with props 1`] = `
<v-layout-stub tag="div">
  <v-flex-stub tag="div" text-xs-center="">
    <blockquote class="blockquote">
      “First, solve the problem. Then, write the code.”
      <footer><small><em>—John Johnson</em></small></footer>
    </blockquote>
  </v-flex-stub>
</v-layout-stub>
`;
